timesketch/lib/utils.py

```diff
@@ -149,6 +190,24 @@ def read_and_validate_csv(file_handle, delimiter=",", mandatory_fields=None):
         delimiter = codecs.decode(delimiter, "utf8")
 
     header_reader = pandas.read_csv(file_handle, sep=delimiter, nrows=0)
+
+    if headersMapping:
+        # sanity check of headersMapping 
+        # e.g., 2 or more mandatory headers are mapped with the same exisiting CSV header
+        res = headers_mapping_sanity_check(header_reader, headersMapping)
+        if res != "OK":
```

Comment:
I would rather prefer if the method returns a true or false if it was okay or not. Comparing strings is not trivial, e.g. someone could use the method and then try to compare agains „ok“ and not „OK“.

Reply:
That's a good consideration. I thought to return an error to allow the user to understand which is the problem with her headers mapping. What about: returning an error (i.e., e msg string) ONLY when any of the sanity checks fail? In this case the if condition would become: `if res:`, thus, no string comparison. What do you think?